### PR TITLE
Rails Capstone Project: Budget Buddy

### PR DESCRIPTION
**Pull request by ikennarichard**
## Rails Capstone Project: Budget Buddy
- Added a splash screen with the name of the app and links to the sign-up and log in pages.
- The user can register in the app with full name, email and password.
- The user can log into the app using email and password.
- Implemented authorizations with `cancancan`, only logged-in user can access pages.
- Implemented authentication with `devise`.
- Used `bullet` for detecting n+1 issues. 
- Added the categories page.
- Implemented form for adding new Categories.
- Added a page for each category's transactions.
- Implemented form for adding new transactions.
- Implement user interaction for the (<) back button.
- Added unit/integration tests with `rspec` and `capybara`.
- [Project description](https://www.loom.com/share/8653d017edfb4ce59b4c79760943112f?sid=9ea18934-5404-432e-9052-17d99fea175d)

### CODE REVIEW

**Review by Ismailco — CHANGES_REQUESTED** [review]
#### Hi @ikennarichard, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!
You did a great job, keep up the good work 👍🏼

giphy


## Highlights

✔️ Setup tests
✔️ Great UI
✔️ Clean code
✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Suggested changes
- [ ] Great work on this project 👏🏼, however, the buttons for `Add transaction` and `Add Category` are not positioned correctly on the bottom, one suggestion to do this is you can try giving them position absolute and bottom 0 then add a margin-bottom for the top container of the hight of the button container.

|||
|-|-|
|Screenshot_20230810_153900|Screenshot_20230810_153855|

- [ ] The `Save` button is not showing on the page, you need to scroll down to see it, kindly make sure to fix this so the `Save` button is showing at the button of the page without any overflow

trans


_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by ikennarichard** [comment]
Thank you @Ismailco !

File: README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply README.md:127]: - Well resolved

File: README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  <h1><b>BudgetBuddy</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+    - [🚀 Live Demo](#live-demo)
```
[the remark this answers is not in the document]
Comment (Whoistolu) [reply README.md:15]: - Well resolved

**Review by Whoistolu — APPROVED** [review]
### Hi @ikennarichard 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.
- [ ] [OPTIONAL] This project has a wrong GITFLOW, please note that:
1. For group capstone projects the correct flow for requesting reviews is Github flow which means dev merging to main.
2. For solo capstone projects, the flow for requesting reviews is Gitflow which means feature branch merging to the development.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ikennarichard** [comment]
Thank you @Whoistolu !

**Pull request merged**